python/nav/web/auth.py

```diff
@@ -77,6 +77,7 @@ class RemoteUserConfigParser(NAVConfigParser):
 enabled=no
 login-url=
 varname=REMOTE_USER
+workaround=default
```

Comment:
I'd rather have the default value named `none`, as the default isn't really a workaround at all.